.github/workflows/workflow.yaml

```diff
@@ -2,13 +2,14 @@ name: OpenGHG tests
 
 on:
   push:
-    branches: [master, devel]
+    branches:
+      - '*'
```

Comment:
Do we want to run the CI on push to all branches?

Reply:
Again I believe this can be reverted for now. If we did want to put this change in that can be on a separate PR